src/components/BooksForm.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const booksCategories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+const BooksForm = () => (
+  <div>
+    <form>
+      <label htmlFor="title">
+        Title
+        <input id="title" name="title" type="text" />
+      </label>
+      <label htmlFor="options">
+        Title
+        <select name="options" id="options">
+          {booksCategories.map((category) => (
```

Comment:
**Required**
- [ ] As required by the last reviewer, you need to create a folder called `containers`, then move the `BookForm.js` and `BookList.js` components into it.
- [ ] Also, it is good to follow instructions as laid down by the project specification. Kindly rename the `BooksForm` to `BookForm` as specified in the spec.